### DISCUSSION TITLE
Menerima webhook dari pengguna lain

### DISCUSSION THREAD

**Discussion by shahib-mirbath** [kirimwa-id/forum#100] [Tanya Jawab]
Selamat pagi.
Setelah mengaktifkan webhook, saya menerima respon dari pengguna lain kirimWA.id. Itu kenapa ya?

**Comment by rioastamal** [comment]
Bisa tolong PM ke email di rio@kirimwa.id detailnya. Hal ini harusnya tidak terjadi.

**Reply by shahib-mirbath** [reply]
Baik. Sudah saya PM email

**Comment by galangex** [comment]
> 

cara mengaktifkan webhook haruskah punya domain/website sendiri bang?

**Reply by rioastamal** [reply]
Betul, URL/endpoint harus bisa diakses dari internet. Tidak harus ada domain.

**Reply by galangex** [reply]
Kalau tidak ada domain, cara biar webhook bisa diakses di internet gimana bang?

**Reply by rioastamal** [reply]
Pakai public IP saja dari server yang digunakan. Kalau untuk testing bisa gunakan ngrok untuk membuka akses localhost ke-internet.
